content/_guides/memo.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Sending memos
+category: using
+weight: 61
+---
+
+Libera.chat supports sending memos to users or channels using MemoServ. Memos
+may be sent to users or channels. You may send memos to offline users for them
+to read when they come online.
+
+Both the sender's nick and the target nick or channel name must be registered.
+See [Nickname registration]({% link _guides/registration.md %}) for more
+information.
+
+## Sending a memo
+
+To send a memo, run the following command in your IRC client:
+
+```
+/msg MemoServ SEND <nick/#channel> <memo text>
+```
+
+If you send a memo to a channel, all users in the channel receive the memo in
```

Comment:
This is incorrect. If the sender can view the access list then the memo is sent to the accounts that have either +O or +o channel flags.

Reply:
@glguy I think I was basing this off of another implementation of MemoServ. I've updated the wording in the article to align more closely with Libera's `HELP` message.